js/tests/unit/tooltip.js

```diff
@@ -14,9 +14,11 @@ $(function () {
       $.fn.bootstrapTooltip = $.fn.tooltip.noConflict()
     },
     afterEach: function () {
+      // Dispose tooltip to make sure all tethers get destroyed after each test
+      $('.tooltip').bootstrapTooltip('dispose')
```

Comment:
I have added dispose to the tooltip tests afterEach so it destroys all the tether instances after each test -- before those tether instances were lingering there and the newly added test was not failing always as it fails only when the `tether#destroy` is called and is the only tether in the page.